pymatgen/io/vasp/inputs.py

```diff
@@ -2275,19 +2275,19 @@ def from_file(filename: str):
 
         functionals = []
         for p in fdata.split("End of Dataset"):
-            if p.strip():
-                single = PotcarSingle(p + "End of Dataset\n")
+            if p_strip := p.strip():
```

Comment:
Ooo walrus is in 3.8?  nice!
![image](https://user-images.githubusercontent.com/14003693/228062123-b9bdd8bb-c3d3-4fa1-8623-bb1b3d081068.png)